tests/tools/eslint-fuzzer.js

```diff
@@ -320,7 +320,10 @@ describe("eslint-fuzzer", function () {
 									node,
 									message: "no foos allowed",
 									fix: fixer =>
-										fixer.replaceText(node, "bar"),
+										fixer.replaceText(
```

Comment:
This change does not seem to be related to the purpose of this PR?

Reply:
Thank you for the explanation.
I think the test would be easier to understand when instead of replacing the `Program` node only the "foo" `ExpressionStatement` is replaced (keeping the comment intact instead of "readding" it).